.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "skewer"]
```

Comment:
I think we should consider using git-subrepo instead.  The main reason is simply that it avoids one extra step for the user.  But let me know if you tried it and prefer submodules.

Reply:
The subrepo-based instructions: https://github.com/skupperproject/skewer#setting-up-skewer-for-your-own-example and https://github.com/skupperproject/skewer#setting-up-skewer-for-your-own-example

Reply:
Changed to git-subrepo, thanks for the instrucctions!